ops/cli.py

```diff
@@ -134,138 +150,153 @@ def _render_ctmpl(ctmpl_file: str, **kwargs) -> int:
+                color="blue"
             )
         )
-        HELM_UPGRADE = f"helm upgrade wfl-k8s gotc-charts/wfl -f {values} --install"
         print(
             dye_msg_with_color(
-                msg=f"[✔] WFL is deployed, run `kubectl get pods` to see details",
-                color="green",
+                msg="=> This must run on a non-split VPN.",
+                color="blue"
+            )
+        )
+        UPGRADE = f"helm upgrade wfl-k8s gotc-charts/wfl -f {values} --install"
+        CLI._subprocess_call(UPGRADE)
+        print(
+            dye_msg_with_color(
+                msg="[✔] WFL is deployed. Run `kubectl get pods` for details.",
+                color="green"
             )
         )
-        return subprocess.call(HELM_UPGRADE, shell=True)
 
     @register
     def deploy(self, arguments):
         """Deploy WFL to Cloud GKE (VPN is required)"""
         parser = argparse.ArgumentParser(description=f"{self.deploy.__doc__}")
+        parser.add_argument(
+            "-b",
+            "--branch",
+            dest="branch",
+            default="master",
+            help="Use this branch of gotc-deploy."
+        )
         parser.add_argument(
             "-e",
             "--environment",
             dest="environment",
             default="dev",
-            choices={"dev",},
-            help="Environment to deploy to",
-        )
-        parser.add_argument(
-            "-v",
-            "--version",
-            dest="version",
-            default="latest",
-            help="Which version of WFL to deploy, i.e. a WFL version string or [latest] by default",
+            choices={"dev"},
+            help="Deploy to this environment."
         )
         parser.add_argument(
             "-n",
             "--namespace",
             dest="namespace",
             default="",
-            help="The namespace you want to use for deployemnt",
+            help="Deploy to this namespace."
+        )
+        parser.add_argument(
+            "-t",
+            "--tag",
+            dest="tag",
+            default="latest",
+            help="Deploy image with this tag in DockerHub."
         )
         args = parser.parse_args(arguments)
-
         CLI._is_available("helm", "kubectl", "gcloud", "git", "docker")
         CLI._set_up_helm()
         CLI._set_up_k8s(namespace=args.namespace)
-
-        with tempfile.TemporaryDirectory(
-            prefix=str(uuid.uuid4()), dir=Path("./")
-        ) as dir_name:
-            CLI._clone_config_repo(
-                dest=Path(f"./{dir_name}/gotc-deploy"), branch="master"
-            )
-
-            file_name = "wfl-values.yaml.ctmpl"
-            shutil.copy(
-                str(Path(f"./{dir_name}/gotc-deploy/deploy/gotc-dev/helm/{file_name}")),
-                Path(f"./{dir_name}/{file_name}"),
-            )
+        with tempfile.TemporaryDirectory() as dir:
+            pwd = os.getcwd()
+            os.chdir(dir)
```

Comment:
Do you need to change dir or can you just clone to this directory directly? If an exception is thrown then would the current directory be in this stale location?

Reply:
Yes, but who cares?
We could wrap it in a try.
This seemed to be the simplest thing that can work.
https://agilefaq.wordpress.com/2007/11/03/what-is-kiss-principle/ :agileaf:

Reply:
Rex already implemented the ability to specify the folder so there was no need for a try or to manually manage system resources.

Reply:
After adding the subprocess reporting, I saw pathnames like "././...".
I figured that was due to things like str(Path(f"…")) getting passed around
This way, we just let the system choose the directory and we run in it.
Otherwise you're bouncing between strings and paths to coerce filenames.
I believe this is simpler.  Restore what was there before and compare.
I'll back out the changes if you still prefer that.  Thanks.

Reply:
OK - when you put it like this this sounds sensible.